doc/changes/changes_2.0.2.md

```diff
@@ -0,0 +1,17 @@
+# Error Code Crawler Maven Plugin 2.0.2, released 2024-??-??
```

Comment:
Do you plan to do a release for this fix?

Reply:
No, as we discussed with @kaklakariada, there are some other issues, which should be added to the new release, so the release will be done later.